packages/react-core/src/components/Toolbar/__tests__/ToolbarToggleGroup.test.tsx

```diff
@@ -124,8 +124,8 @@ describe('Toolbar', () => {
         <ToolbarContent>{items}</ToolbarContent>
       </Toolbar>
     );
-    expect(view.find('#save-button').exists());
-    expect(view.find('#clear-button').exists());
+    expect(screen.findByTestId('save-button')).toBeTruthy();
+    expect(screen.findByTestId('clear-button')).toBeTruthy();
     expect(view).toMatchSnapshot();
```

Comment:
In RTL the return value from render is an object, and we need to access the value of `container` on that object for an accurate snapshot.

```suggestion
    expect(view.container).toMatchSnapshot();
```

Reply:
Updated. I think that was also why the snapshot had doubled content because it was also reporting the value of the `container`.